ptvsd/wrapper.py

```diff
@@ -602,6 +602,7 @@ def __init__(self, socket, pydevd, logfile=None, killonclose=True):
         self.modules_mgr = ModulesManager(self)
         self.disconnect_request = None
         self.launch_arguments = None
+        self.launch_options = {}
```

Comment:
I'd check this to debug options. For instance when attaching to Django application we'll need to option DJANGO_DEBUG=True in the attach request for brains to work.